page/dafault_page.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gxpage
+
+// DefaultPage is the default implementation of Page interface
+type DefaultPage struct {
```

Comment:
I wonder will user implement their own Page struct, 
- if yes, suggest rename interface `Page` to `Pager`, rename `DefaultPage` to `Page` which will be frequently, so the name should not keep simple
- if no,  remove the interface `Page`, rename `DefaultPage` to `Page`